File: src/main/dart/net/coderextreme/data/AllenBox.dart
// Copyright 2021 by John Carlson.  All rights reserved.
import 'dart:io';
import '../x3d.dart';
var X3D0 =  X3D(

      profile_ : SFString('Immersive'),
      version_ : SFString('4.0'),
      head_ : 
        head(
          meta_ : [
            meta(
              name_ : SFString('title'),
              content_ : SFString('AllenBox.x3d')),

            meta(
              name_ : SFString('creator'),
              content_ : SFString('Allen Box')),

            meta(
              name_ : SFString('created'),
              content_ : SFString('8 July 2025')),

            meta(
              name_ : SFString('license'),
              content_ : SFString('../license.html'))]),
      Scene_ : 
        Scene(
          children_ : [
            WorldInfo(
              title_ : SFString('AllenBox.x3d')),

            NavigationInfo(
              type_ : MFString([SFString("EXAMINE")]),
              avatarSize_ : MFFloat([SFFloat(0.15), SFFloat(1.53), SFFloat(0.75)]),
              speed_ : 0.5),

            Transform(
              DEF_ : SFString('Floor'),
              scale_ : SFVec3f([SFDouble(1), SFDouble(0.0125), SFDouble(1)]),
              translation_ : SFVec3f([SFDouble(0), SFDouble(-0.0125), SFDouble(0)]),
              child_ : 
                Shape(
                  geometry_ : 
                    Box(),
                  appearance_ : 
                    Appearance(
                      material_ : 
                        Material())))]));
void main() { exit(0); }

File: src/main/c/net/coderextreme/data/AllenBox.c
#include "C:/x3d-code/www.web3d.org/x3d/languages/c/Concretes.h"
void main(int argc, char ** argv) {
Browser browser = X3D.getBrowser();
X3D X3D0;
X3D0.profile = "Immersive";
X3D0.version = "4.0";
head head1 = createNode("head");
meta meta2 = createNode("meta");
meta2.name = "title";
meta2.content = "AllenBox.x3d";
head1.meta = new MFNode();

head1.meta[0] = meta2;

meta meta3 = createNode("meta");
meta3.name = "creator";
meta3.content = "Allen Box";
head1.meta[1] = meta3;

meta meta4 = createNode("meta");
meta4.name = "created";
meta4.content = "8 July 2025";
head1.meta[2] = meta4;

meta meta5 = createNode("meta");
meta5.name = "license";
meta5.content = "../license.html";
head1.meta[3] = meta5;

head = head1;

WorldInfo WorldInfo7 = createNode("WorldInfo");
WorldInfo7.title = "AllenBox.x3d";
children = new MFNode();

children[0] = WorldInfo7;

NavigationInfo NavigationInfo8 = createNode("NavigationInfo");
NavigationInfo8.avatarSize = new MFFloat(new float[0.15,1.53,0.75]);
NavigationInfo8.speed = 0.5;
NavigationInfo8.type = new MFString(new java.lang.String["EXAMINE"]);
children[1] = NavigationInfo8;

Transform Transform9 = createNode("Transform");
Transform9.DEF = "Floor";
Transform9.scale = new SFVec3f(new float[1,0.0125,1]);
Transform9.translation = new SFVec3f(new float[0,-0.0125,0]);
Shape Shape10 = createNode("Shape");
Box Box11 = createNode("Box");
Shape10.geometry = Box11;

Appearance Appearance12 = createNode("Appearance");
Material Material13 = createNode("Material");
Appearance12.material = Material13;

Shape10.appearance = Appearance12;

Transform9.child = new undefined();

Transform9.child[0] = Shape10;

children[2] = Transform9;

}
